src/REL/d/t/d_t_map_inst.cpp

```diff
@@ -4,5 +4,17 @@ SPECIAL_ACTOR_PROFILE(TAG_MAP_INST, dTgMapInst_c, fProfile::TAG_MAP_INST, 0x021d
 
 int dTgMapInst_c::create() {
     field_0xfc = params & 0xff;
```

Comment:
for something like this, is it worth it to change the field to be something like `params1_0xFF` just so we know it can be linked to object parameters